documentation/src/main/asciidoc/public/reference/_explicit_backend_index_operations.adoc

```diff
@@ -0,0 +1,13 @@
+[[explicit-backend-index-operations]]
+= Explicit backend/index operations
+:leveloffset: +1
+
+include::_explicit_operations_apply-analazyer-normalyzer.adoc[]
```

Comment:
Typo in the file name.

That being said, splitting this into multiple files seems a little bit extreme... no?